src/AerospikeCache.php

```diff
@@ -0,0 +1,135 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+
+class AerospikeCache extends AbstractAdapter
+{
+    private const WRAPPER_NAME = 'data';
+
+    /** @var \Aerospike */
+    private $aerospike;
+
+    /** @var string */
+    private $namespace;
+
+    /** @var string */
+    private $set;
+
+    public function __construct(
+        \Aerospike $aerospike,
+        string $namespace,
+        string $set,
+        string $cacheNamespace = '',
+        int $defaultLifetime = 0
+    ) {
+        $this->aerospike = $aerospike;
+        $this->namespace = $namespace;
+        $this->set = $set;
+        parent::__construct($cacheNamespace, $defaultLifetime);
+    }
+
+    protected function doFetch(array $ids): array
+    {
+        $result = [];
+
+        $keys = $this->initializeKeysForAerospike($ids);
+
+        $this->aerospike->getMany($keys, $records);
+
+        foreach ($records as $record) {
```

Comment:
It seems following code from this line below is not covered by any test - so fetching the items is not tested. Or am I missing something?

Reply:
It should be tested in `testShouldReadExistingRecordsFromAerospike`..